docs/mailbox-controller.md

```diff
@@ -0,0 +1,24 @@
+# Mailbox Controller
+
+This is a component in an architecture not yet shared in detail.  It is an early milestone in edge multicluster, achieving partial functionality and implemented in a simple crude approach that is layered on top of transparent multicluster.  In this approach the edge-mc layer maintains a kcp workspace, called a mailbox, per selected edge destination per EdgePlacement object.  Then TMC syncs between the mailbox workspaces ad the edge destinations.
+
```

Comment:
typo ?  "between the mailbox workspaces **and** the edge"

Reply:
Good catch.  Fixed in latest.